### PR TITLE
Forward traffic from 3 addition ports for telco bastion

### PR DESCRIPTION
**Pull request by johnbieren**
I'd like to be able to forward traffic from a pod in openshift ci through the bastion service for the provision host and three separate masters, so I am adding three more ports here.

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

**Comment by johnbieren** [comment]
@stevekuznetsov Can you take a look at this?

File: core-services/sshd-bastion/telco/deployment.yaml [hunk 1]
```diff
@@ -77,4 +83,4 @@ items:
             secretName: sshd-host-keys
         - name: config
           configMap:
-            name: sshd-config
\ No newline at end of file
+            name: sshd-config
```
Comment (yuvalk) [inline core-services/sshd-bastion/telco/deployment.yaml:86]: no need for that \n

File: core-services/sshd-bastion/telco/sshd_config [hunk 2]
```diff
@@ -7,5 +7,8 @@ PasswordAuthentication no
 PermitTunnel yes
 PidFile /tmp/ssh.pid
 Port 2222
+Port 2223
+Port 2224
+Port 2225
 StrictModes no
-GatewayPorts yes
\ No newline at end of file
+GatewayPorts yes
```
Comment (yuvalk) [inline core-services/sshd-bastion/telco/sshd_config:14]: no need for that \n

File: core-services/sshd-bastion/telco/deployment.yaml (hunk 1 above)
Comment (johnbieren) [reply core-services/sshd-bastion/telco/deployment.yaml:86]: vim is set to do that for me to follow POSIX standards

**Pull request merged**